index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="status-bar">
+            <p class="hour">9:41</p>
+            <div class="icons">
+                <div class="network">
+                    <img src="images/toolbar_icons/connection.png" alt="mobile connection icon">
+                </div>
+                <div class="wifi">
+                    <img src="images/toolbar_icons/wifi.png" alt="wifi icon">
+                </div>
+                <div class="battery">
+                    <img src="images/toolbar_icons/battery.png" alt="battery icon">
+                </div>
+            </div>
+        </div>
+        <div class="menu">
+            <h6 class="name">John Doe</h6>
+            <button type="button" aria-label="open close main menu" class="toolbar-icon">
+                <img src="images/toolbar_icons/menu.png" alt="">
+            </button>
+        </div>
+    </header>
+    <main>
+        <section class="headline">
+            <h1 class="title">
+                Hey There. I’m Alex
+            </h1>
+            <h3 class="subtitle">
+                I’am a Software Developer
+            </h3>
+            <p class="description">
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <div class="social-media">
+                <a href="#">
+                    <img class="social-media-icon" src="images/social_media_icons/github.png" alt="">
+                </a>
+                <a href="#">
+                    <img class="social-media-icon" src="images/social_media_icons/linkedin.png" alt="">
+                </a>
+                <a href="#">
+                    <img class="social-media-icon" src="images/social_media_icons/angelist.png" alt="">
+                </a>
+                <a href="#">
+                    <img class="social-media-icon" src="images/social_media_icons/twitter.png" alt="">
+                </a>
+                <a href="#">
+                    <img class="social-media-icon" src="images/social_media_icons/medium.png" alt="">
+                </a>
+            </div>
+            <div class="illustration">
+                <img src="images/illustrations/illustration-header-mobile.svg" alt="">
+            </div>
```

Comment:
Kindly use the CSS background-image class to set the headline image according to the requirement. It is always a good design pattern to always have design images set as background with CSS.